xml/chrony.xml

```diff
@@ -207,9 +207,12 @@
    use specific server names or IP addresses, for example:
   </para>
 
-<screen>server 0.europe.pool.ntp.org
-server 1.europe.pool.ntp.org
-server 2.europe.pool.ntp.org</screen>
+<screen>
```

Comment:
Please don't insert line breaks after `<screen>` and before `</screen>`, even if they are stripped out by the stylesheets.